content/en/tracing/service_catalog/integrations.md

```diff
@@ -4,15 +4,53 @@ kind: documentation
 further_reading:
 - link: "/tracing/service_catalog/service_definition_api/"
   tag: "Documentation"
-  text: "Service Definition API"
+  text: "Learn about the Service Definition API"
+- link: "/integrations/opsgenie/"
+  tag: "Documentation"
+  text: "Learn about the OpsGenie integration"
+- link: "/integrations/pagerduty/"
+  tag: "Documentation"
+  text: "Learn about the PagerDuty integration"
 ---
 
+## Overview
+
+When you configure a service account for a [Datadog integration][10], you can incorporate metadata from your integrations into service definitions in the [Service Catalog][9]. You can also utilize autocomplete and validation features when editing a service definition in an [integrated development environment (IDE)](#ide-integrations).
+
+## OpsGenie integration
+
+You can add OpsGenie metadata to a service so that the [Service Catalog][11] displays and links to information such as who is on-call for the service.
```

Comment:
```suggestion
You can add OpsGenie metadata to a service so that the Service Catalog displays and links to information such as who is on-call for the service.
```